### DISCUSSION TITLE
Unable to connect github copilot

### DISCUSSION THREAD

**Discussion by 0dayoverflow** [community/community#120781] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am working with visual studio 2022 and when I write code I don't get the predictions and in the terminal I get this : Copilot 1.156.0.0 (v1.156.0.0@82fafb995)

GitHub Copilot Agent 1.156.0 (1.156.0)
[DEBUG] [agent] [2024-04-22T21:10:28.919Z] Agent service starting
[DEBUG] [agent] [2024-04-22T21:10:29.350Z] Telemetry initialized
Auth Status: OK
Successfully authorized GitHub Copilot.
[DEBUG] [getCompletions] [2024-04-22T21:10:58.819Z] Requesting completion at position 19:11, between "\r\n asd" and "\r\n\r\n".
[INFO] [default] [2024-04-22T21:10:59.384Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[DEBUG] [getCompletions] [2024-04-22T21:10:59.924Z] Requesting completion at position 19:13, between "\r\n async" and "\r\n\r\n".
[INFO] [ghostText] [2024-04-22T21:11:00.005Z] Cancelled during debounce
[DEBUG] [getCompletions] [2024-04-22T21:11:00.196Z] Requesting completion at position 19:16, between "\r\n abstract" and "\r\n\r\n".
[INFO] [default] [2024-04-22T21:11:00.286Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
statusNotification: Warning: unable to verify the first certificate
[ERROR] [ghostText] [2024-04-22T21:11:00.392Z] Error on ghost text request: (FetchError) unable to verify the first certificate
[ERROR] [certificates] [2024-04-22T21:11:00.393Z] Your current Copilot license doesn't support proxy connections with custom certificates. Please visit https://gh.io/copilot-network-errors to learn more. Original cause: {"type":"system","_name":"FetchError","code":"UNABLE_TO_VERIFY_LEAF_SIGNATURE"}
[INFO] [RequestProposalsAsync] 0 completions available.
[DEBUG] [getCompletions] [2024-04-22T21:11:00.703Z] Requesting completion at position 19:9, between "\r\n p" and "\r\n\r\n".
[DEBUG] [getCompletions] [2024-04-22T21:11:00.751Z] Requesting completion at position 19:15, between "\r\n partial" and "\r\n\r\n".
[INFO] [ghostText] [2024-04-22T21:11:00.843Z] Cancelled during debounce
[DEBUG] [getCompletions] [2024-04-22T21:11:00.932Z] Requesting completion at position 19:12, between "\r\n prop" and "\r\n\r\n".
[INFO] [ghostText] [2024-04-22T21:11:01.014Z] Cancelled during debounce

Added to a message "Your current Copilot license doesn't support proxy connections with custom certificates" and my copilot license is personal and from github pro that added to the fact that I don't belong to any company. what can be a possible solution? Thanks.
![imagen](https://github.com/community/community/assets/98193746/22f07d43-85f9-42b9-be2e-d9364ed4719c)

**Comment by ayim** [comment]
Closing as duplicate, continuing convo here: https://github.com/orgs/community/discussions/120698

**Reply by haliovilla** [reply]
I'm experiencing the same issue. The strangest thing of all is that GitHub Copilot works fine in Visual Studio Code and even GitHub Copilot Chat works fine within Visual Studio Community 2022.
